backend/src/nodes/image_nodes.py

```diff
@@ -135,10 +135,15 @@ def run(
 
         os.makedirs(base_directory, exist_ok=True)
 
-        status, buf_img = cv2.imencode(f".{extension}", img)
-        with open(full_path, "wb") as outf:
-            bytes_written = outf.write(buf_img)
-            status = status and bytes_written == len(buf_img)
+        # Write image with opencv if path is ascii, since imwrite doesn't support unicode
+        # This saves us from having to keep the image buffer in memory, if possible
+        if full_path.isascii():
```

Comment:
What if on windows try to convert path in UTF-8?
https://github.com/opencv/opencv/issues/4292#issuecomment-601686965

Reply:
![image](https://user-images.githubusercontent.com/34788790/187781341-65f3a2d8-d346-4774-adec-2d639c2cd8f0.png)